src/regtech_api_commons/oauth2/oauth2_admin.py

```diff
@@ -96,3 +96,9 @@ def associate_to_lei(self, user_id: str, lei: str) -> None:
     def associate_to_leis(self, user_id: str, leis: Set[str]):
         for lei in leis:
             self.associate_to_lei(user_id, lei)
+
+    def delete_group(self, lei: str) -> Dict[str, Any] | None:
+        try:
+            return self._admin.delete_group(lei)
+        except kce.KeycloakError:
+            return None
```

Comment:
I would raise the exception using RegTechHttpException so that there is at least some indicator of what happened.

Reply:
Oh, yes! Redoing it.